packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -52,6 +52,20 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 - Added [PrintApi](/docs/api/pos-ui-extensions/apis/print-api) and a [PrintPreview](/docs/api/pos-ui-extensions/components/printpreview) component.
       `,
     },
+    {
+      type: 'Generic',
+      anchorLink: '2024101',
+      title: '2024.10.1',
+      sectionContent: `
+- Added in POS version: 9.22
+- Removed in POS version: N/A
+- Release day: 11/11/2024.
```

Comment:
Is this accurate still?

Reply:
Yes it is. We can cut 2024.10.1 anytime this week, I'll try to get to it before Friday